docs/whirlpool/docs/03-Whirlpools SDK/02-Whirlpool Management/01-Create Pool.md

```diff
@@ -0,0 +1,94 @@
+2. **Initial Price**: Set the initial price of token 1 in terms of token 2.
+3. **Funder**: This will be your wallet, which will fund the initialization process.
+4. **Create Instructions**: Use the `createSplashPoolInstructions()` function to generate the required pool creation instructions.
+    ```tsx
+    import { createSplashPoolInstructions } from '@orca-so/whirlpools'
+
+    const { poolAddress, instructions, initializationCost } = await createSplashPoolInstructions(
+        Rpc,
+        tokenMintOne,
+        tokenMintTwo,
+        initialPrice,
+        wallet
+    );
+    ```
+5. Submit Transaction: Include the generated pool creation instructions in a Solana transaction and send it to the network using the Solana SDK.
+
+### Creating Concentrated Liquidity Pools
+
+Concentrated Liquidity Pools offer more flexibility:
+
+1. **Token Mint Addresses**: Provide the two token mints.
+2. **Tick Spacing**: Set the tick spacing, which defines the intervals for price ticks. Visit [this link](https://orca-so.gitbook.io/orca-developer-portal/whirlpools/interacting-with-the-protocol/orca-whirlpools-parameters#initialized-feetier-and-tickspacing) to learn more about the available values of tick spacing and their corresponding fee rates.
+3. **Initial Price**: Specify the initial price of token 1 in terms of token 2.
+4. **Funder**: This will be your wallet, which will fund the pool initialization.
+5. **Create instructions**: Use the `createConcentratedLiquidityPool()` function to create the pool.
+    ```tsx
+    import { createConcentratedLiquidityPool } from '@orca-so/whirlpools'
+
+    const { poolAddress, instructions, initializationCost } = await createConcentratedLiquidityPool(
+        Rpc,
+        tokenMintOne,
+        tokenMintTwo,
+        tickSpacing,
+        initialPrice,
+        wallet
+    );
+    ```
+6. **Submit Transaction**: Include the generated pool creation instructions in a Solana transaction and send it to the network using the Solana SDK.
+
+### Comparison
+| Feature            | Splash Pools       | Concentrated Liquidity Pools     |
+| ------------------ | ------------------ | -------------------------------- |
+| Complexity         | Low                | High                             |
+| Initial Parameters | Token mints, price | Token mints, tick spacing, price |
+| Capital Efficiency | Moderate           | High                             |
+| Ideal For          | Beginners          | Advanced Users                   |
+
+## 3. Usage Examples
+
+### Launching a Token Pair with a Splash Pool
+
+Suppose you want to launch a new memecoin and pair it with USDC. You can use `createSplashPoolInstructions()` to set up the pool quickly with a given initial price. This is ideal if you want to keep things simple and start earning trading fees with minimal configuration. For example, if a development team is building a launchpad for memecoins, Splash Pools are an ideal solution.
```

Comment:
Since the functions will be different on different SDKs (rust, legacy, ts) would it make sense to just say: "You can use Splash Pools to set a up the pool ..."

Reply:
Good point, I will apply this to all docs.